pom.xml

```diff
@@ -32,7 +32,7 @@
 
 		<environment>${ENV_VAR}</environment>
 		<target-properties>target/classes/application.properties</target-properties>
-		<source-properties>src/main/environment/common_${environment}.properties</source-properties>
+		<source-properties>target/classes/common_${environment}.properties</source-properties>
```

Comment:
@drtechie Can you help me understand why this path change is required? Is it for mvn-antrun-plugin to concatenate?

Reply:
If we set ENV_VAR=ci

1. `common_ci.properties` is taken first by the mvn-resources-plugin and filtered.
2. In the filtering step, all @@ variables are replaced with environment variables.
3. The filter common_ci.properties is output to target/classes/common_ci.properties.
4. `target/classes/common_ci.properties` is then merged with `application.properties`

This was the only way I could find to make mvn-antrun-plugin and mvn-resources-plugin work together.

Reply:
@drtechie Thankyou for the clarification.